src/transport/raw/tests/NetworkTestHelpers.cpp

```diff
@@ -29,6 +29,10 @@ namespace Test {
 
 CHIP_ERROR IOContext::Init()
 {
+#if defined(CHIP_DEVICE_LAYER_TARGET_OPEN_IOT_SDK)
```

Comment:
Was this error due to the chip stack being already initialized ?

Reply:
The issue was that we needed to initialize the stack and SDK timer for the correct operation of the tests.